.vscode/settings.json

```diff
@@ -1,78 +1,67 @@
-// Place your settings in this file to overwrite default and user settings.
 {
-    "files.associations": {
-        "*.podspec": "ruby",
-        "Podfile": "ruby",
-        "*.h": "objective-c",
-        "*.js": "javascriptreact",
-        "*.snap": "javascriptreact",
-        ".stylelintrc": "json"
-    },
-    "eslint.enable": false,
-    "flow.enabled": false,
-    "editor.formatOnSave": true,
-    "prettier.singleQuote": false,
-    "prettier.trailingComma": "es5",
-    "prettier.typescriptEnable": [
-        "typescript",
-        "typescriptreact"
-    ],
-    "prettier.semi": false,
-    "prettier.printWidth": 120,
-    "files.trimTrailingWhitespace": true,
-    "editor.renderWhitespace": "boundary",
-    "editor.rulers": [
-        120
-    ],
-    "editor.tabSize": 2,
-    "files.exclude": {
-        "**/.git": true,
-        "**/.svn": true,
-        "**/.hg": true,
-        "**/.DS_Store": true,
-        ".jest/": true,
-        "build": true,
-        "Example/build": true,
-        "Example/DerivedData": true,
-        "Example/Pods/Headers/**": true
-    },
-    "search.exclude": {
-        "**/node_modules": true,
-        "externals/**": true,
-        "**/typings/**": true,
-        "Pod/Assets/Emission.*": true,
-        ".vscode/.react/*": true
-    },
-    "clang.compilationDatabase": "${workspaceRoot}/Example/compile_commands.json",
-    "typescript.tsdk": "./node_modules/typescript/lib",
-    "spellchecker.ignoreWordsList": [
-        "orta",
-        "sarah",
-        "luc",
-        "mzikherman",
-        "CHANGELOG",
-        "Eigen",
-        "scrollview",
-        "blankspace",
-        "ESLint",
-        "GeneVC",
-        "keychain",
-        "linting",
-        "inline",
-        "iPad",
-        "iPhone",
-        "ListView",
-        "ARTemporaryAPIModule",
-        "Inbox",
-        "matt"
-    ],
-    "stylelint.additionalDocumentSelectors": [
-        "typescript",
-        "typescriptreact"
-    ],
-    "todohighlight.exclude": [
-        "node_modules/**",
-        ".vscode/**",
-        "externals/**"
-    ]
+  "files.associations": {
+    "*.podspec": "ruby",
+    "Podfile": "ruby",
+    "*.h": "objective-c",
+    "*.js": "javascriptreact",
+    "*.snap": "javascriptreact",
+    ".stylelintrc": "json"
+  },
+  "eslint.enable": false,
+  "flow.enabled": false,
+  "editor.formatOnSave": true,
+  "prettier.singleQuote": false,
+  "prettier.trailingComma": "es5",
+  "prettier.typescriptEnable": ["typescript", "typescriptreact"],
+  "prettier.semi": false,
+  "prettier.printWidth": 120,
+  "files.trimTrailingWhitespace": true,
+  "editor.renderWhitespace": "boundary",
+  "editor.rulers": [120],
+  "editor.tabSize": 2,
+  "files.exclude": {
+    "**/.git": true,
+    "**/.svn": true,
+    "**/.hg": true,
+    "**/.DS_Store": true,
+    ".jest/": true,
+    "build": true,
+    ".awcache": true,
+    ".happypack": true,
+    "Example/build": true,
```

Comment:
```diff
+    ".awcache": true,
+    ".happypack": true,
```

is the real change here, but I removed the comments so prettier can handle the JSON formatting